### PR TITLE
Build with dotnet (#70).

### PR DESCRIPTION
**Pull request by cesarbs**
#70 

cc @victorhurdugaci @Tratcher

### CODE REVIEW

File: test/Microsoft.AspNet.WebSockets.Client.Test/project.json [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+    "Microsoft.AspNet.WebSockets.Server": "1.0.0-*",
+    "Microsoft.AspNet.Server.Kestrel": "1.0.0-*",
+    "Microsoft.AspNet.Testing": "1.0.0-*",
+    "xunit": "2.1.0"
+  },
+  "testRunner": "xunit",
+  "frameworks": {
+    "dnx451": {
+      "dependencies": {
+        "xunit.runner.console": "2.1.0"
```
Comment (victorhurdugaci) [inline test/Microsoft.AspNet.WebSockets.Client.Test/project.json]: you can move this in the top level deps node

File: build.sh [hunk 2]
```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
```
Comment (victorhurdugaci) [inline build.sh]: I don't think this is the latest version of this file. see the templates in universe

Comment (cesarbs) [inline build.sh]: Fixed.

**Comment by cesarbs** [comment]
I can't get `AutobahnTestClient` to run with dotnet. Will continue investigating tomorrow.

**Comment by victorhurdugaci** [comment]
Is this done?

File: src/Microsoft.AspNetCore.WebSockets.Client/project.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
     "Microsoft.AspNetCore.WebSockets.Protocol": "0.1.0-*"
   },
   "frameworks": {
+    "dnx451": {},
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.WebSockets.Client/project.json]: Shouldn't be required.

File: src/Microsoft.AspNetCore.WebSockets.Protocol/project.json [hunk 4]
```diff
@@ -10,6 +10,7 @@
     "keyFile": "../../tools/Key.snk"
   },
   "frameworks": {
+    "dnx451": {},
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.WebSockets.Protocol/project.json]: Dont' need.

File: src/Microsoft.AspNetCore.WebSockets.Server/project.json [hunk 5]
```diff
@@ -15,6 +15,7 @@
     "Microsoft.Extensions.Options": "1.0.0-*"
   },
   "frameworks": {
+    "dnx451": {},
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.WebSockets.Server/project.json]: Don't need.

**Comment by davidfowl** [comment]
Alomst. Remove all of the dnx451.

**Comment by cesarbs** [comment]
Build passes on Windows and Linux (if you look at the Travis job). Fails on Mac due to an issue in dotnet. Can I merge or should I wait until it builds everywhere?

**Comment by cesarbs** [comment]
@Tratcher Could you take a look at the changes I've done to AutobahnTestClient?

**Comment by Tratcher** [comment]
They look fine.

**Pull request merged**